### PR TITLE
NOD & SC | Check PDF encryption on backend

### PR DESCRIPTION
**Pull request by Mottie**
## Summary

- _(Summarize the changes that have been made to the platform)_
  > Previously the frontend would check the PDF for an encrypted signature, which exists for PDFs locked with either an owner or user password. The backend PDF library is better equipped to detect PDFs locked with a user-only password.
- _(If bug, how to reproduce)_
- _(What is the solution, why is this the solution)_
- _(Which team do you work for, does your team own the maintenance of this component?)_
- _(If using a flipper, what is the end date of the flipper being required/success criteria being targeted)_

## Related issue(s)

[#77084](https://github.com/department-of-veterans-affairs/va.gov-team/issues/77084)

## Testing done

- _Describe what the old behavior was prior to the change_
  > Previously the frontend would check the PDF for an encrypted signature, which exists for PDFs locked with either an owner or user password. The backend PDF library is better equipped to detect PDFs locked with a user-only password.
- _Describe the steps required to verify your changes are working as expected_
  > Once the backend work from [#74493](https://github.com/department-of-veterans-affairs/va.gov-team/issues/74493) is available in production, we can merge and test this work in staging.
- _Describe the tests completed and the results_
  > Updated unit tests
- _Exclusively stating 'Specs and automated tests passing' is NOT acceptable as appropriate testing
- _Optionally, provide a link to your [test plan](https://depo-platform-documentation.scrollhelp.site/developer-docs/create-a-test-plan-in-testrail) and [test execution records](https://depo-platform-documentation.scrollhelp.site/developer-docs/execute-tests-in-testrail)_

## Screenshots

upload password protected PDF now uploads the file to the backend first, when a missing user password error is returned, the PDF password input is shown. An incorrect password was initially provided showing an incorrect password error. Then the correct password was entered and the PDF password has been added message is then shown

## What areas of the site does it impact?

Notice of Disagreement & Supplemental Claim

### Quality Assurance & Testing

- [x] I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [ ] Linting warnings have been addressed
- [ ] Documentation has been updated ([link to documentation](#) \*if necessary)
- [x] Screenshot of the developed feature is added
- [ ] [Accessibility testing](https://depo-platform-documentation.scrollhelp.site/developer-docs/wcag-2-1-success-criteria-and-foundational-testing) has been performed

### Error Handling

- [ ] Browser console contains no warnings or errors.
- [ ] Events are being sent to the appropriate logging solution
- [ ] Feature/bug has a monitor built into Datadog or Grafana (if applicable)

### Authentication

- [ ] Did you login to a local build and verify all authenticated routes work as expected with a test user

### :warning: Team Sites (only applies to modifications made to the VA.gov header) :warning:

- [ ] The vets-website header does not contain any web-components
- [ ] I used the [proxy-rewrite steps](https://github.com/department-of-veterans-affairs/vets-website/tree/main/src/applications/proxy-rewrite#local-dev) to test the injected header scenario
- [ ] I reached out in the `#sitewide-public-websites` Slack channel for questions

## Requested Feedback

(OPTIONAL) _What should the reviewers know in addition to the above. Is there anything specific you wish the reviewer to assist with. Do you have any concerns with this PR, why?_

### CODE REVIEW

**Review by anniebtran — COMMENTED** [review]
Reviewed this with Rocio — looks great overall, just one small nit: clean up naming of test

File: src/applications/appeals/shared/tests/utils/focus.unit.spec.js [hunk 1]
```diff
@@ -188,6 +186,117 @@ describe('focusFileCard', () => {
...
+            secondary
+            class="vads-u-padding-x--0 vads-u-padding-y--1"
+            label="Add another file evidence to be reviewed by the Board"
+            text="Add another file"
+          />
+        </div>
+      </div>,
+    );
+
+  it('should focus on add another button', async () => {
```
[the remark this answers is not in the document]
Comment (Mottie) [reply src/applications/appeals/shared/tests/utils/focus.unit.spec.js]: Fixed, thanks!

**Review by anniebtran — APPROVED** [review]
LGTM

**Review by rmessina1010 — APPROVED** [review]
LGTM

**Pull request merged**